src/main/java/at/unisalzburg/apted/distance/AllPossibleMappingsTED.java

```diff
@@ -21,36 +21,36 @@
  * SOFTWARE.
  */
 
-package distance;
+package at.unisalzburg.apted.distance;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Iterator;
-import node.Node;
-import node.NodeIndexer;
-import costmodel.CostModel;
+import at.unisalzburg.apted.node.Node;
+import at.unisalzburg.apted.node.NodeIndexer;
+import at.unisalzburg.apted.costmodel.CostModel;
 
 /**
  * Implements an exponential algorithm for the tree edit distance. It computes
  * all possible TED mappings between two trees and calculated their minimal
  * cost.
  *
- * @param C type of cost model.
- * @param D type of node data.
+ * @param <C></C> type of cost model.
```

Comment:
This change causes params to disappear from the generated javadoc. I think it should be <C> and <D> without the closing tags.

Reply:
well spotted